cdc/sink/codec/json.go

```diff
@@ -451,6 +457,14 @@ func (d *JSONEventBatchEncoder) AppendRowChangedEvent(e *model.RowChangedEvent)
 		message.Schema = &e.Table.Schema
 		message.Table = &e.Table.Table
 
+		if message.Metadata == nil {
+			message.Metadata = &messageMeta{}
+		}
+
+		messageMeta := message.Metadata.(*messageMeta)
+		messageMeta.tables = append(messageMeta.tables, fmt.Sprintf("`%s`.`%s`", e.Table.Schema, e.Table.Table))
```

Comment:
what about using a map, since table names could be duplicated

Reply:
I think it would be better if the user knows which table each record comes from and how long each record is. So we need to store an ordered list instead of a set of table names,.